nautobot/utilities/utils.py

```diff
@@ -907,7 +907,9 @@ def get_filterset_parameter_form_field(model, parameter, filterset=None):
 
         form_field = DynamicModelMultipleChoiceField(**form_attr)
     elif isinstance(field, ContentTypeMultipleChoiceFilter):
-        plural_name = model._meta.verbose_name_plural
+        from nautobot.utilities.utils import slugify_dashes_to_underscores
```

Comment:
Why are we importing from `nautobot.utilities.utils` in `nautobot.utilities.utils`? :-)

Reply:
Because we need utils in our utils!

Reply:
I'm a genius. Don't question it.